todo/src/components/Task.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { toggleComplete } from '../actions';
+import { connect } from 'react-redux';
+
+class Task extends Component {
+
+  toggleComplete(event) {
+    this.props.toggleComplete(this.props.index);
+  }
+
+  render() {
+    return (
+      <li>
+        <input
+          type="checkbox"
+          // onChange={this.toggleComplete.bind(this)}
+          />
+        {this.props.todo.value}
+      </li>
+    );
+  }
+}
+
+export default connect(null, { toggleComplete })(Task);
```

Comment:
🥇 Nice catch with null -- no need for mapStateToProps the way your component is set up.